### ISSUE TITLE
Help: Error when starting container: `failed executing "iptables --policy INPUT DROP"` 

### ISSUE THREAD

**Issue by mikeyjarvis19** [qdm12/gluetun#400]
**Is this urgent?**: No

**Host OS** (approximate answer is fine too): Raspberry Pi OS (Kernel version 5.10.11-v7l+)

**CPU arch** or **device name**: Raspberry Pi 4 (4GB)

**What VPN provider are you using**: PIA

**What is the version of the program** (See the line at the top of your logs)

```
Running version latest built on 2021-03-03T01:33:15Z (commit 83b5a94)
```

**What's the problem** 🤔

Getting error when starting the container: 
```
2021/03/05 22:35:21 ERROR cannot enable firewall: failed executing "iptables --policy INPUT DROP": iptables v1.8.4 (legacy): can't initialize iptables table `filter': Table does not exist (do you need to insmod?)
```
Have tried to follow the steps in the `Setup` section, and get the following outputs when trying the suggested commands:
```shell 
$ insmod /lib/modules/tun.ko
insmod: ERROR: could not load module /lib/modules/tun.ko: No such file or directory

$ modprobe tun
modprobe: ERROR: ../libkmod/libkmod.c:586 kmod_search_moddep() could not open moddep file '/lib/modules/5.10.11-v7l+/modules.dep.bin'
modprobe: FATAL: Module tun not found in directory /lib/modules/5.10.11-v7l+
```

**Share your logs... (careful to remove in example tokens)**
```log
=========================================
gluetun    | ================ Gluetun ================
gluetun    | =========================================
gluetun    | ==== A mix of OpenVPN, DNS over TLS, ====
gluetun    | ======= Shadowsocks and HTTP proxy ======
gluetun    | ========= all glued up with Go ==========
gluetun    | =========================================
gluetun    | =========== For tunneling to ============
gluetun    | ======== your favorite VPN server =======
gluetun    | =========================================
gluetun    | === Made with â�¤ï¸�  by github.com/qdm12 ====
gluetun    | =========================================
gluetun    | 
gluetun    | Running version latest built on 2021-03-03T01:33:15Z (commit 83b5a94)
gluetun    | 
gluetun    | 
gluetun    | ðŸ”§  Need help? https://github.com/qdm12/gluetun/issues/new
gluetun    | ðŸ’»  Email? quentin.mcgaw@gmail.com
gluetun    | â˜•  Slack? Join from the Slack button on Github
gluetun    | ðŸ’¸  Help me? https://github.com/sponsors/qdm12
gluetun    | 2021/03/05 21:39:36 INFO OpenVPN version: 2.4.10
gluetun    | 2021/03/05 21:39:36 INFO Unbound version: 1.10.1
gluetun    | 2021/03/05 21:39:36 INFO IPtables version: v1.8.4
gluetun    | 2021/03/05 21:39:36 INFO Settings summary below:
gluetun    | |--OpenVPN:
gluetun    |    |--Verbosity level: 1
gluetun    |    |--Run as root: enabled
gluetun    |    |--Provider:
gluetun    |       |--Private Internet Access settings:
gluetun    |          |--Network protocol: udp
gluetun    |          |--Regions: uk london
gluetun    |          |--Encryption preset: strong
gluetun    |          |--Custom port: 0
gluetun    | |--DNS:
gluetun    |    |--Plaintext address: 1.1.1.1
gluetun    |    |--DNS over TLS:
gluetun    |       |--Unbound:
gluetun    |           |--DNS over TLS providers:
gluetun    |               |--cloudflare
gluetun    |           |--Listening port: 53
gluetun    |           |--Access control:
gluetun    |               |--Allowed:
gluetun    |                   |--0.0.0.0/0
gluetun    |                   |--::/0
gluetun    |           |--Caching: enabled
gluetun    |           |--IPv4 resolution: enabled
gluetun    |           |--IPv6 resolution: disabled
gluetun    |           |--Verbosity level: 1/5
gluetun    |           |--Verbosity details level: 0/4
gluetun    |           |--Validation log level: 0/2
gluetun    |           |--Blocked hostnames:
gluetun    |           |--Blocked IP addresses:
gluetun    |               |--127.0.0.1/8
gluetun    |               |--10.0.0.0/8
gluetun    |               |--172.16.0.0/12
gluetun    |               |--192.168.0.0/16
gluetun    |               |--169.254.0.0/16
gluetun    |               |--::1/128
gluetun    |               |--fc00::/7
gluetun    |               |--fe80::/10
gluetun    |               |--::ffff:0:0/96
gluetun    |           |--Allowed hostnames:
gluetun    |       |--Block malicious: enabled
gluetun    |       |--Update: every 24h0m0s
gluetun    | |--Firewall:
gluetun    | |--System:
gluetun    |    |--Process user ID: 1000
gluetun    |    |--Process group ID: 1000
gluetun    |    |--Timezone: NOT SET âš ï¸� - it can cause time related issues
gluetun    | |--HTTP control server:
gluetun    |    |--Listening port: 8000
gluetun    |    |--Logging: enabled
gluetun    | |--Public IP getter:
gluetun    |    |--Fetch period: 12h0m0s
gluetun    |    |--IP file: /tmp/gluetun/ip
gluetun    | |--Github version information: enabled
gluetun    | 2021/03/05 21:39:36 INFO storage: merging by most recent 6933 hardcoded servers and 6933 servers read from /gluetun/servers.json
gluetun    | 2021/03/05 21:39:36 INFO routing: default route found: interface eth0, gateway 192.168.32.1
gluetun    | 2021/03/05 21:39:36 INFO routing: local subnet found: 192.168.32.0/20
gluetun    | 2021/03/05 21:39:36 INFO routing: default route found: interface eth0, gateway 192.168.32.1
gluetun    | 2021/03/05 21:39:36 INFO routing: adding route for 0.0.0.0/0
gluetun    | 2021/03/05 21:39:36 INFO firewall: firewall disabled, only updating allowed subnets internal list
gluetun    | 2021/03/05 21:39:36 INFO routing: default route found: interface eth0, gateway 192.168.32.1
gluetun    | 2021/03/05 21:39:36 INFO openvpn configurator: checking for device /dev/net/tun
gluetun    | 2021/03/05 21:39:36 WARN TUN device is not available: open /dev/net/tun: no such file or directory
gluetun    | 2021/03/05 21:39:36 INFO openvpn configurator: creating /dev/net/tun
gluetun    | 2021/03/05 21:39:36 INFO firewall: enabling...
gluetun    | 2021/03/05 21:39:36 ERROR cannot enable firewall: failed executing "iptables --policy INPUT DROP": iptables v1.8.4 (legacy): can't initialize iptables table `filter': Table does not exist (do you need to insmod?)
gluetun    | Perhaps iptables or your kernel needs to be upgraded.: exit status 3
gluetun    | 2021/03/05 21:39:36 INFO Shutdown successful
```

**What are you using to run your container?**: Docker Compose

Please also share your configuration file:

```yml
version: "3.7"

services:
  gluetun:
    image: qmcgaw/gluetun:latest
    container_name: gluetun
    environment:
      - VPNSP=private internet access
      - REGION=UK London
      - OPENVPN_USER=${VPN_USER}
      - OPENVPN_PASSWORD=${VPN_PASSWORD}
    volumes:
      - /home/pi/synced/gluetun:/gluetun
    cap_add:
      - NET_ADMIN
    restart: unless-stopped
```

Device `tun0` is present when I run `ifconfig`, have installed all available updates on the base machine, including kernel updates. Seems like more of an issue with the base machine than my config for the image, but not really sure. Have also done some googling around the error + trying to run the above commands on a pi, but haven't found anything that helps. This is the first time I've tried to use this image.

**Comment by qdm12** [comment]
Let's first focus on that error 

```
iptables v1.8.4 (legacy): can't initialize iptables table `filter': Table does not exist (do you need to insmod?)
```

On your host what happens if you do (list iptables rules):

```sh
sudo iptables -nvL
```

Does this work or fail?

Maybe this [is a raspberry pi related issue](https://askubuntu.com/a/1130316)? Let me know the outcome, I'll add the resolution to the wiki!

**Comment by mikeyjarvis19** [comment]
`sudo iptables -nvL` seemed to work (let me know if the output would be of any use, and I'll post it here). Running the `rpi-update` commands in https://askubuntu.com/a/1130316 seemed to do the trick, the container is starting up okay now I've run those updates. Have also managed to connect another container and verify I'm behind the VPN.  Thank you!

**Comment by qdm12** [comment]
Ah great! I've added it to the Wiki.

Fixed in 829ad632f130b04a02693c5a587aa45386bed156
